e2e_playwright/st_dataframe_selections_test.py

```diff
@@ -12,23 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import platform
 
 import pytest
 from playwright.sync_api import Locator, Page, expect
 
 from e2e_playwright.conftest import ImageCompareFunction, wait_for_app_run
-from e2e_playwright.shared.app_utils import expect_prefixed_markdown
+from e2e_playwright.shared.app_utils import COMMAND_KEY, expect_prefixed_markdown
 from e2e_playwright.shared.dataframe_utils import (
     calc_middle_cell_position,
     select_column,
     select_row,
     sort_column,
 )
 
-# Meta = Apple's Command Key; for complete list see https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_key_values#special_values
-_COMMAND_KEY = "Meta" if platform.system() == "Darwin" else "Control"
```

Comment:
moved this to our utils so that we can share it between tests (now also used in the `st_dialog` test)